examples/basics/data_row_metadata.ipynb

```diff
@@ -3,7 +3,9 @@
   {
```

Comment:
<p>Do we want to expose all of this? It seems like a lot of text that isn't relevant to the user. Maybe just show only one, similar to the below sections?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/587/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I will clear the text output of this cell. I do want to tell users that they can list all available metadata fields in SDK</p>